core/src/main/java/io/questdb/cairo/CairoEngine.java

```diff
@@ -298,10 +298,10 @@ public void completeSnapshot() throws SqlException {
             boolean keepLock
     ) {
         securityContext.authorizeTableCreate();
-        return createTableInsecure(securityContext, mem, path, ifNotExists, struct, keepLock, true);
+        return createTableAsRoot(securityContext, mem, path, ifNotExists, struct, keepLock, true);
     }
 
-    public @NotNull TableToken createTableInsecure(
+    public @NotNull TableToken createTableAsRoot(
```

Comment:
nit: root is confusing as it also means db root sometimes
```suggestion
    public @NotNull TableToken createTableAsSu(
```

Reply:
fixed in https://github.com/questdb/questdb/pull/3790/commits/653b0d402b8b293906dcbb02b8a0ada2a1a151b5